configuration/configure-automatic-upgrades.adoc

```diff
@@ -1,18 +1,21 @@
 [id="configure-automatic-upgrades"]
-= Configuring automatic upgrades for secured clusters
+= Configuring automatic upgrades for manifest-installed secured clusters
 include::modules/common-attributes.adoc[]
 :context: configure-automatic-upgrades
 
 toc::[]
 
 [role="_abstract"]
-You can automate the upgrade process for each secured cluster and view the upgrade status from the {product-title-short} portal.
+If you installed {product-title} by using the manifest installation method, also known as the `roxctl` CLI method, you can automate the upgrade process for each secured cluster. This process is also called the _legacy installation method_ in the {product-title-short} portal. You can also view the upgrade status from the {product-title-short} portal.
 
-Automatic upgrades make it easier to stay up-to-date by automating the manual task of upgrading each secured cluster.
+[NOTE]
+====
+Automatic upgrades are only available for {product-title-short} systems that were installed by using the manifest installation method. If you installed {product-title-short} by using the Operator, upgrades are controlled by using {olm-first}. If you installed {product-title-short} by using Helm charts, you must use Helm to upgrade.
+====
 
-With automatic upgrades, after you upgrade Central;  Sensor, Collector, and Compliance services in all secured clusters, automatically upgrade to the latest version.
+Automatic upgrades make it easier to stay up-to-date by automating the manual task of upgrading each secured cluster. If you have automatic upgrades enabled, the upgrader will upgrade the entire secured cluster to the same version as Central.
```

Comment:
This block is perfectly fine. If we want to add an important detail, I would suggest this version:

```suggestion
Automatic upgrades make it easier to stay up-to-date by automating the manual task of upgrading each secured cluster. If you have automatic upgrades enabled and the secured cluster is configured for receiving automated upgrades, the upgrader will upgrade the entire secured cluster to the same version as Central.
```